<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>160</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="toolTip">
   <string>herp derp
derp lerp</string>
  </property>
  <widget class="QLabel" name="card_text">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>115</y>
     <width>150</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string>Card Text</string>
   </property>
  </widget>
  <widget class="QLabel" name="card_image">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>150</width>
     <height>120</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>120</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>120</height>
    </size>
   </property>
   <property name="text">
    <string>Card Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="card_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>175</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>175</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>175</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Card Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
